### PR TITLE
storage: Assorted improvements

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
Grr, this was supposed to help with the TestStoragePartitions.testPartitions flake, but it doesn't...

File: pkg/storaged/utils.js [hunk 1]
```diff
@@ -749,7 +749,7 @@ export function teardown_active_usage(client, usage) {
             // If we would remove all physical volumes of a volume
             // group, remove the whole volume group instead.
             if (pvs.length == client.vgroups_pvols[p].length) {
-                return vg.Delete({ 'tear-down': { t: 'b', v: true } });
+                return vg.Delete({ 'tear-down': { t: 'b', v: true } }).then(reload_systemd);
```
Comment (cockpituous) [inline pkg/storaged/utils.js:752]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220914-124338-048b9d83-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/vdo-details.jsx [hunk 2]
```diff
@@ -171,7 +171,7 @@ export class VDODetails extends React.Component {
             }
 
             function wipe_with_teardown(block) {
-                return block.Format("empty", { 'tear-down': { t: 'b', v: true } });
+                return block.Format("empty", { 'tear-down': { t: 'b', v: true } }).then(reload_systemd);
```
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:174]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220914-124338-048b9d83-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/utils.js (hunk 1 above)
Comment (cockpituous) [inline pkg/storaged/utils.js:752]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220914-140853-d27bc977-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/vdo-details.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:174]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220914-140853-d27bc977-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by mvollmer** [comment]
I'll try to increase the coverage a little bit here.

File: pkg/storaged/vdo-details.jsx (hunk 2 above)
Comment (mvollmer) [reply pkg/storaged/vdo-details.jsx:174]: This is impossible to reach on fedora-36, which doesn't support VDO.  (Or does it?)

Comment (mvollmer) [reply pkg/storaged/vdo-details.jsx:174]: (No, it doesn't.)

Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:174]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220915-111445-72bee719-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by martinpitt** [comment]
Retrying [debian-testing flake](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220920-135055-ded8fcc5-debian-testing/log.html#320)

**Review by martinpitt — APPROVED** [review]
Danke!

**Comment by martinpitt** [comment]
The [debian failure](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220923-071829-ded8fcc5-debian-testing /log.html) is persistent..

**Comment by mvollmer** [comment]
I'll debug this in CI a bit.

**Comment by mvollmer** [comment]
Test runs fine in isolation in rhos: https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220923-092005-c4c57e47-debian-testing/log.html

**Comment by mvollmer** [comment]
Success on centosci: https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220923-093623-4a64b39b-debian-testing/log.html

**Comment by mvollmer** [comment]
Success also on rhos: https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220923-104142-4a64b39b-debian-testing/log.html

**Comment by mvollmer** [comment]
No ideas, let's try again and maybe increase the timeout.

**Review by martinpitt — APPROVED** [review]
Re-acking in case it goes green now. I suppose there's some noise due to the "3x affected" retries?

File: pkg/storaged/vdo-details.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/storaged/vdo-details.jsx:174]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17736-20220923-131055-4172823d-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Pull request merged**